docs/modules/servers/pages/distributed/operate/metrics.adoc

```diff
@@ -95,25 +105,35 @@ This is for instance how the JMAP dashboard looks like:
 
 image::metrics.png[metrics for the JMAP protocol request latencies]
 
-== Running and configuring Grafana
+== Running and configuring Prometheus/Grafana
```

Comment:
I think you should leave this section intact, as it is describing how to run Grafana with the ES metric exporter solution, that some people might still use, and create a new section for Prometheus/Grafana